standalone/standalone.py

```diff
@@ -1678,6 +1678,8 @@ def shutdown_vllm(process: subprocess.Popen, timeout: int = 20):
             usable_cpu_count = multiprocessing.cpu_count() // 2
         max_workers = usable_cpu_count
 
+    # modify model_list to ignore any jsonl files present in the directory
+    models_list = [model for model in models_list if model.endswith(".jsonl") != True]
```

Comment:
Strange `if not model.endswith(".jsonl")]` was not propaged here, did you update manually?

Reply:
https://github.com/redhat-et/ilab-on-ocp/pull/117

Reply:
That is odd, I must have skipped a `pipeline.py gen-standalone`. Good catch thanks!